prober/http.go

```diff
@@ -247,6 +332,27 @@ func ProbeHTTP(ctx context.Context, target string, module config.Module, registr
 	if resp == nil {
 		resp = &http.Response{}
 	}
+	i := 0
+	for _, trace := range tt.traces {
+		// We get the duration for the first request from chooseProtocol
```

Comment:
You should log the times too, so the user can get back to the individual request times

Reply:
I've added logging the raw timestamps which is probably most useful.

Reply:
I think the diffs would be more useful to a human.